pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -864,11 +864,15 @@ async def _handle_model_response(
             else:
                 tool_calls.append(part)
 
-        if texts:
+        # at the moment, we prioritize at least executing tool calls if they are present
+        # in the future, we'd consider making this configurable at the agent or run level
+        # this accounts for cases like anthropic returns that might contain a text response
+        # and a tool call response, where the text response just indicates the tool call
```

Comment:
```suggestion
        # At the moment, we prioritize at least executing tool calls if they are present.
        # In the future, we'd consider making this configurable at the agent or run level.
        # This accounts for cases like anthropic returns that might contain a text response
        # and a tool call response, where the text response just indicates the tool call will happen.
```
I found it a little hard to read without punctuation, not knowing if the next line was a continuation of the previous line or not